### ISSUE TITLE
autoOpen without switching to the new tab

### ISSUE THREAD

**Issue by anglimath** [gdh1995/vimium-c#1017]
Is it possible to "autoOpen" selected text without switching to the new tab? 

Similar question for Vomnibar, how do I open a bookmark in background tab like what "LinkHints.activateOpenInNewTab" does?

**Comment by gdh1995** [comment]
Please try `map xxx autoOpen reuse="curent"` and you may also add other necessary parameters to this mapping.

If you do want to open a bookmark, you may assign it a unique title and then use:
```
map xxx openBookmark title="<the-unique-title>" reuse="new-bg"
```

**Comment by anglimath** [comment]
My updates on question 1, 

Sorry for the ambiguity in my previous question. I am trying to use autoOpen to "open the selected text in this tab (tab 1)" in a background tab (say tab 2) and stay in (tab 1). Similar to the behavior of "LinkHints.activateOpenInNewTab". 

I tried the script 'map xxx autoOpen reuse="current"', however, it opens the selected text in place (it replaces tab 1). 

I looked into your code and I believe 'map xxx autoOpen reuse="new-bg"' is exactly what I need. 

Now question 2, my question is as follows

Is it possible for me to implement a different version of "Vomnibar.activateBookmarksInNewTab" which opens the bookmark in a background tab, just like the difference between "LinkHints.activateOpenInNewForegroundTab" and "LinkHints.activateOpenInNewTab"? 

I tried reuse="new-bg" on both "Vomnibar.activateBookmarksInNewTab" and "Vomnibar.activateBookmarks", however, it seems both don't work as expected. reuse="new-bg" does not work on Vomnibar commands. 

Anyway, thank you very much for this amazing tool.
